gatsby-node.js

```diff
@@ -13,3 +14,5 @@ exports.onCreateWebpackConfig = ({ actions }) => {
     },
   })
 }
+
+exports.createPages = implementations.createPages
```

Comment:
```suggestion
exports.createPages = async (blabla) => {
  await  createPages(blabla)
}
```
To bych nevyrovnal.
`exports.createPages` je public interface a` implementations.createPages` je implementace.